src/general/OptionsMenu.cs

```diff
@@ -973,4 +980,12 @@ private void OnCustomUsernameTextChanged(string text)
 
         UpdateResetSaveButtonState();
     }
+
+    private void OnOpenScreenshotFolder()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+
+        OS.ShellOpen(ProjectSettings.GlobalizePath(Constants.SCREENSHOT_FOLDER));
+        UpdateResetSaveButtonState();
```

Comment:
Isn't it unnecessary to call this as this button doesn't change the options state?

Reply:
I think this is the last problem with this.
I merged master into this branch, that apparently removed the one BOM change? Next time you work on this you should pull first before starting working.